lib/config.js

```diff
@@ -19,24 +20,37 @@ if(process.env.MONGO_PORT) {
     mongo_port = process.env.MONGO_PORT
 }
 
-if(process.env.MONGO_USE_SSL.toLowerCase() == "true") {
-    use_ssl = true
+if(process.env.MONGO_USE_SSL) {
+    if(process.env.MONGO_USE_SSL.toLowerCase() == "true") {
+        use_ssl = true
+    }
 }
 
-if(process.env.MONGO_VALIDATE_SSL.toLowerCase() == "false") {
-    validate_ssl = false
+if(process.env.MONGO_VALIDATE_SSL) {
+    if(process.env.MONGO_VALIDATE_SSL.toLowerCase() == "false") {
+        validate_ssl = false
+    }
 }
 
 if(process.env.MONGO_AUTH_USER && process.env.MONGO_AUTH_PWD) {
     auth_details = `${process.env.MONGO_AUTH_USER}:${process.env.MONGO_AUTH_PWD}@`
 }
 
+var hosts = service_host.split(',')
+
+for (let i=0; i<hosts.length;i++) {
+  connection_details += `${hosts[i]}:${mongo_port},`
+}
+
+connection_details = connection_details.replace(/,\s*$/, "");
```

Comment:
Do we expect extra spaces after the comma?

Reply:
No extra spaces are expected after the comma. I thought it would be a good idea to "sanitize" the input though. Removing the trailing comma should be enough.